src/components/Asset/AssetActions/Pool/Add/index.tsx

```diff
@@ -40,15 +41,25 @@ export default function Add({
   const [newPoolShare, setNewPoolShare] = useState('0')
   const [isWarningAccepted, setIsWarningAccepted] = useState(false)
 
+  // eslint-disable-next-line security/detect-non-literal-regexp
+  const maxDecimalsValidation = new RegExp(
```

Comment:
we also have this defined in `Publish/_validations.ts` so we should abstract this to somewhere, guess if in doubt drop into generic `@utils/constants.ts` :-)